WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyControlTileViewModel.cs

```diff
@@ -60,6 +70,15 @@ protected override void OnActivated(CompositeDisposable disposables)
 		PrivacyBar?.Activate(disposables);
 	}
 
+	private static bool IsProgressVisible(decimal privateAndSemiPrivateBalance, decimal totalBalance)
+	{
+		var hasPrivacy = privateAndSemiPrivateBalance > 0;
+		var hasBalance = totalBalance > 0;
+		var everythingIsPrivate = privateAndSemiPrivateBalance == totalBalance;
+		var isProgressVisible = hasPrivacy && hasBalance && !everythingIsPrivate;
```

Comment:
I don't get the point of `everythingIsPrivate`. First the naming is invalid, if the total balance is semi-private it would be true which is incorrect. Second, we want to display the bar even if it is full private.

Reply:
OK, I used the word "private" for both semi-private and full-fullprivate. AFAIK, we don't have a term to include semi and full. My fault :) It would be useful to have a consensus. Eg. If we have 2 BTC that are semi-private + 3 that are private, we have 5 BTC that a degree of privacy (some are semi, but others are full). So what's the term for those 5 BTC? 

Anyways, I've fixed the code and remove that condition. Please check if now it's correct.
Thanks!

Reply:
>  So what's the term for those 5 BTC?

There isn't really one, usually we call as you did "Private and Semi-private"